core/Cargo.toml

```diff
@@ -35,7 +35,7 @@ iroha_version = { version = "=2.0.0-pre.1", path = "../version", features = ["wa
 iroha_actor = { path = "../actor" }
 iroha_config = { version = "=2.0.0-pre.1", path = "../config" }
 iroha_futures = { path = "../futures" }
-iroha_telemetry = { path = "../telemetry", optional = true }
+iroha_telemetry = { path = "../telemetry" }
```

Comment:
are we sure it'll no longer be optional?

Reply:
Same as Core. We rely on the status endpoint in production code, so metrics (and thus telemetry) are now a hard requirement.